x-pack/plugins/lists/common/schemas/types/non_empty_nested_entries_array.ts

```diff
@@ -12,7 +12,8 @@ import { entriesMatchAny } from './entry_match_any';
 import { entriesMatch } from './entry_match';
 import { entriesExists } from './entry_exists';
 
-export const nestedEntriesArray = t.array(t.union([entriesMatch, entriesMatchAny, entriesExists]));
+export const nestedEntries = t.union([entriesMatch, entriesMatchAny, entriesExists]);
```

Comment:
this seems like a correct naming convention here given the existing schema name...